### ISSUE TITLE
Fights always result in a defeated message despite winning

### ISSUE THREAD

**Issue by XarsYs** [Mezeman1/idle-ant-farm#36]
I have all my army stats so high that the enemy in some areas can't do damage to me, and when I fight I always get the message that I was defeated by the bug. I still get loot (at least most of the times), the bug's HP does drop, the armies stays the same as expected.

**Comment by Mezeman1** [comment]
Hmm, any steps to reproduce?

**Comment by XarsYs** [comment]
It just always happened, and from the start of the game. My save seems to have deleted itself, but it was a guest account, and all the way from the first battle it was like that. I just tried from the start again, all I did was get 24 ants, got into the first battle, and the bug is already present. I also get achievements for defeating them.

Chrome Version 129.0.6668.58 (Official Build) (64-bit)

This is the console output, if it helps:
index.8008cd90.js:45 Battle Loop Started
index.8008cd90.js:45 Battle Started
index.8008cd90.js:45 Bug Wins!
index.8008cd90.js:45 Regeneration after defeat finished
index.8008cd90.js:45 Battle loop stopped

![image](https://github.com/user-attachments/assets/5e84c81b-2cda-465d-a958-b8456693db1c)
